Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:alpine AS build
+
+WORKDIR /opt/python-gitlab
+COPY . .
+RUN python setup.py bdist_wheel
+
+FROM python:alpine
+
+WORKDIR /opt/python-gitlab
+COPY --from=build /opt/python-gitlab/dist dist/
+RUN pip install $(find dist -name *.whl)
```

Comment:
We could also delete the wheel here, and build the docker image with [`--squash`](https://docs.docker.com/engine/reference/commandline/build/#squash-an-images-layers---squash-experimental), to save a tiny bit of extra space.

Reply:
`--squash` still seems to be experimental, but otherwise good idea!